app/styles/app.css

```diff
@@ -41,10 +41,26 @@ a {
   margin: 10px;
 }
 
+.spec-reminder-item {
+  display: flex;
+  flex-direction: row;
+}
+.spec-reminder-item p {
+  margin-left: 5px;
+}
+
 .add-reminder-btn {
   width: 100px;
 }
 
 .btn:hover {
   cursor: pointer;
 }
+
+.reminder-status {
+  color: green;
+}
+
+.tay-tay {
```

Comment:
@Tman22 just go with it.  It's the easiest way for @mlimberg and I to remember what the class name means...